xml/Country.xml

```diff
@@ -972,553 +972,553 @@
             </name>
         </codelist-item>
         <codelist-item>
             <code>GB</code>
             <name>
-                <narrative>UNITED KINGDOM OF GREAT BRITAIN AND NORTHERN IRELAND (THE)</narrative>
+                <narrative>United Kingdom of Great Britain and Northern Ireland (the)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>US</code>
             <name>
-                <narrative>UNITED STATES OF AMERICA (THE)</narrative>
+                <narrative>United States of America (the)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>UM</code>
             <name>
-                <narrative>UNITED STATES MINOR OUTLYING ISLANDS (THE)</narrative>
+                <narrative>United States Minor Outlying Islands (the)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>UY</code>
             <name>
-                <narrative>URUGUAY</narrative>
+                <narrative>Uruguay</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>UZ</code>
             <name>
-                <narrative>UZBEKISTAN</narrative>
+                <narrative>Uzbekistan</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>VU</code>
             <name>
-                <narrative>VANUATU</narrative>
+                <narrative>Vanuatu</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>VE</code>
             <name>
-                <narrative>VENEZUELA (BOLIVARIAN REPUBLIC OF)</narrative>
+                <narrative>Venezuela (Bolivarian Republic of)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>VN</code>
             <name>
-                <narrative>VIET NAM</narrative>
+                <narrative>Viet Nam</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>VG</code>
             <name>
-                <narrative>VIRGIN ISLANDS (BRITISH)</narrative>
+                <narrative>Virgin Islands (British)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>VI</code>
             <name>
-                <narrative>VIRGIN ISLANDS (U.S.)</narrative>
+                <narrative>Virgin Islands (U.S.)</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>WF</code>
             <name>
-                <narrative>WALLIS AND FUTUNA</narrative>
+                <narrative>Wallis and Futuna</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>EH</code>
             <name>
-                <narrative>WESTERN SAHARA</narrative>
+                <narrative>Western Sahara*</narrative>
```

Comment:
Explanation for the asterisk here: https://www.iso.org/obp/ui/#iso:code:3166:EH

I’ll remove it.